migrations/20220322001917_address.ts

```diff
@@ -0,0 +1,47 @@
+/*****
+ License
+ --------------
+ Copyright © 2020 Mojaloop Foundation
+ The Mojaloop files are made available by the Mojaloop Foundation under the Apache License, Version 2.0 (the 'License') and you may not use these files except in compliance with the License. You may obtain a copy of the License at
+ http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, the Mojaloop files are distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ Contributors
+ --------------
+ This is the official list of the Mojaloop project contributors for this file.
+ Names of the original copyright holders (individuals or organizations)
+ should be listed with a '*' in the first column. People who have
+ contributed from an organization can be listed under the organization
+ that actually holds the copyright for their contributions (see the
+ Gates Foundation organization for an example). Those individuals should have
+ their names indented and be marked with a '-'. Email address can be added
+ optionally within square brackets <email>.
+ * Gates Foundation
+ - Name Surname <name.surname@gatesfoundation.com>
+
+ - Kevin Leyow <kevin.leyow@modusbox.com.com>
+
+ --------------
+ ******/
+import { Knex } from 'knex'
+
+export async function up (knex: Knex): Promise<void | Knex.SchemaBuilder> {
+  return knex.schema.hasTable('Scope')
+    .then((exists: boolean): Knex.SchemaBuilder | void => {
+      if (exists) {
+        return knex.schema.alterTable('Scope', function (table) {
+          table.renameColumn('address', 'address')
+        })
+      }
+    })
+}
+
+export function down (knex: Knex): Promise<void | Knex.SchemaBuilder> {
+  return knex.schema.hasTable('Scope')
```

Comment:
This is meant to reverse whatever the migration was, but I'm not sure what that is, so I can't tell if this is correct